src/MassTransit/Logging/DiagnosticHeaders.cs

```diff
@@ -15,19 +15,32 @@ public static class DiagnosticHeaders
         public const string InputAddress = "input-address";
         public const string ConversationId = "conversation-id";
         public const string TrackingNumber = "tracking-number";
-        public const string CorrelationConversationId = "correlation-conversation-id";
 
         public const string SourceHostMachine = "source-host-machine";
-        public const string SourceHostProcessId = "source-host-process-id";
-        public const string SourceHostFrameworkVersion = "source-host-framework-version";
-        public const string SourceHostMassTransitVersion = "source-host-masstransit-version";
         public const string MessageTypes = "message-types";
 
+        public const string ServiceKind = "span.kind";
+        public const string PeerAddress = "peer.address";
+        public const string PeerHost = "peer.host";
```

Comment:
This should be peer.hostname 
https://github.com/opentracing/specification/blob/master/semantic_conventions.md